engine/pom.xml

```diff
@@ -327,6 +329,11 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
```

Comment:
@kthoms Supporting the Nashorn Engine during testing is fine for me, but adding it as a bundled dependency might be unnecessary for most of the users?